.github/workflows/ci.yml

```diff
@@ -0,0 +1,287 @@
+    - name: Display Go version
+      run: go version
+    - name: Run tests
+      run: make test-v2
+      env:
+        KAFKA_TOPICS_TEST_ZK_ADDR: zookeeper:2181
+        KAFKA_TOPICS_TEST_KAFKA_ADDR: kafka1:9092
+
+    services:
+      zookeeper:
+        image: wurstmeister/zookeeper
+        ports:
+          - "2181:2181"
+
+      kafka1:
+        image: wurstmeister/kafka:2.12-2.4.1
+        ports:
+          - "9092:9092"
+        env:
+          KAFKA_BROKER_ID: 1
+          KAFKA_BROKER_RACK: zone1
+          KAFKA_ADVERTISED_HOST_NAME: kafka1
+          KAFKA_ADVERTISED_PORT: 9092
+          KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+
+      kafka2:
+        image: wurstmeister/kafka:2.12-2.4.1
+        ports:
+          - "9093:9092"
+        env:
+          KAFKA_BROKER_ID: 2
+          KAFKA_BROKER_RACK: zone1
+          KAFKA_ADVERTISED_HOST_NAME: kafka2
+          KAFKA_ADVERTISED_PORT: 9092
+          KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+
+      kafka3:
+        image: wurstmeister/kafka:2.12-2.4.1
+        ports:
+          - "9094:9092"
+        env:
+          KAFKA_BROKER_ID: 3
+          KAFKA_BROKER_RACK: zone2
+          KAFKA_ADVERTISED_HOST_NAME: kafka3
+          KAFKA_ADVERTISED_PORT: 9092
+          KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+
+      kafka4:
+        image: wurstmeister/kafka:2.12-2.4.1
+        ports:
+          - "9095:9092"
+        env: 
+          KAFKA_BROKER_ID: 4
+          KAFKA_BROKER_RACK: zone2
+          KAFKA_ADVERTISED_HOST_NAME: kafka4
+          KAFKA_ADVERTISED_PORT: 9092
+          KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+
+      kafka5:
+        image: wurstmeister/kafka:2.12-2.4.1
+        ports:
+          - "9096:9092"
+        env: 
+          KAFKA_BROKER_ID: 5
+          KAFKA_BROKER_RACK: zone3
+          KAFKA_ADVERTISED_HOST_NAME: kafka5
+          KAFKA_ADVERTISED_PORT: 9092
+          KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+
+      kafka6:
+        image: wurstmeister/kafka:2.12-2.4.1
+        ports:
+          - "9097:9092"
+        env: 
+          KAFKA_BROKER_ID: 6
+          KAFKA_BROKER_RACK: zone3
+          KAFKA_ADVERTISED_HOST_NAME: kafka6
+          KAFKA_ADVERTISED_PORT: 9092
+          KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+  
+  
+  snyk:
+    runs-on: ubuntu-latest
+    needs: [test010, test241]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Run Snyk to check for vulnerabilities
+      uses: snyk/actions/golang@master
+      env:
+        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+      with:
+        args: --severity-threshold=high --fail-on=upgradable
+  
+  
+  publish-ghcr:
+    needs: [setup, snyk]
+    runs-on: ubuntu-latest
+    if: ${{ ( github.ref_type == 'branch' ) && (( github.ref_name == github.master_branch ) || ( github.ref_name == 'v0' )) }}
+    steps:
+    - uses: actions/checkout@v3
+    - name: setup env variables
+      id: vars
+      run: |
+        echo "SHORT_SHA=`echo ${GITHUB_SHA} | cut -c1-8`" >> $GITHUB_ENV
```

Comment:
Can you use `${GITHUB_SHA:0:8}` instead of the pipe to cut

Reply:
resolved in https://github.com/segmentio/topicctl/pull/124/commits/53f831a8b0db1659fd144859f59bf46661f7249c